ovirt/fakeovirt_deployment.yml

```diff
@@ -27,13 +27,9 @@ spec:
       - name: imageio
         image: localhost:5001/imageio:latest
         ports:
-        - containerPort: 12345
-        command: ["/bin/bash"]
-        args:
-        - -c
-        - ovirt-imageio& sleep 15 && 
-          ovirt-imageioctl add-ticket myticket.json &&
-          tail -n+1 -f /var/log/ovirt-imageio/daemon.log
+        - containerPort: 80
+        command: ["/app/entrypoint.py", "/images/cirros.raw", "--ticket-id", "cirros"]
```

Comment:
The `"/app/entrypoint.py"` part should not be necessary, that's the container entrypoint, is never overridden by the command.

Actually, having a `CMD` in the Dockerfile would allow to completely skip the `command` here, if you opt for that option.

Reply:
yeah good point, I was focusing too much on this command instead of `CMD` in the image.
And in the `command` the prefix `/app/entrypoint.py"` does not work but in the `CMD` works